docusaurus/docs/dev-docs/testing.md

```diff
@@ -11,7 +11,7 @@ tags:
 # Unit testing
 
 :::strapi
```

Comment:
```suggestion
:::info
```

Reply:
Suggesting this because "strapi" callouts are purely to higlight attention on important cross-links or other (internal/external) resources for readers.

Also, why are we removing these links? Is their content outdated too?

Reply:
@pwizla no problem, I'll make that change. 
Also yes, the links directly mimic the docs and they are outdated since they do not have Strapi 5 versions from what I saw.